spec/features/state_file/complete_intake_spec.rb

```diff
@@ -10,6 +10,10 @@
     allow_any_instance_of(Routes::StateFileDomain).to receive(:matches?).and_return(true)
   end
 
+  after do
```

Comment:
This will reset sessions, cleaning out the pool of sessions and remove any session information such as cookies between each test

Reply:
do we need this for all feature specs?